Update README.md

rimoliga opened the pull request:
Play Store versions of Termux are deprecated. Changed to F-Droid version. More information in https://github.com/termux/termux-app#google-play-store-deprecated

rimoliga committed:
Update README.md

Play Store versions of Termux ar deprecated. Changed to F-Droid version. Mor information in https://github.com/termux/termux-app#google-play-store-deprecated

prusnak commented:
Isn't it better to link to https://termux.dev/ so people can choose whether they install via GitHub releases or F-Droid?

rimoliga commented:
Yes, it's a valid point.

prusnak commented:
@rimoliga will you update the PR?

rimoliga committed:
Update README.md

Changed link to https://termux.dev/ as suggested in https://github.com/ggerganov/llama.cpp/pull/680#issuecomment-1492977742

rimoliga commented:
@prusnak Yes, it's ready now

rimoliga reviewed (commented):
changed as suggested in https://github.com/ggerganov/llama.cpp/pull/680#issuecomment-1492977742
